src/com/google/javascript/jscomp/AbstractCommandLineRunner.java

```diff
@@ -2681,6 +2681,13 @@ public CommandLineConfig setJsonStreamMode(JsonStreamMode mode) {
       return this;
     }
 
+    private boolean renaming = true;
```

Comment:
I think we don't need this, since we set the values on the CompilerOptions directly in CommandLineRunner.